Cargo.toml

```diff
@@ -20,3 +19,8 @@ exclude = ["tests"]
 [profile.release]
 lto = true
 opt-level = 3
+
+[patch.crates-io]
+oasis-std = { path = "oasis-std" }
+oasis-test = { path = "oasis-test" }
+borsh = { git = "https://github.com/nhynes/borsh" }
```

Comment:
why `nhynes/borsh` and not `oasislabs/borsh`?

Reply:
we can move it to `oasislabs` if you and @peterjgilbert think that's appropriate

Reply:
was just curious if there was a reason. My preference is to have reference a fork in the oasislabs organization

Reply:
Agreed, we should fork in oasislabs org.